src/bypasses/ouo.js

```diff
@@ -0,0 +1,24 @@
+import BypassDefinition from "./BypassDefinition.js"
+
+export default class Ouo extends BypassDefinition {
+    constructor() {
+        super()
+    }
```

Comment:
```suggestion
```
You don't need to call constructor super() as that is the default provided by js. [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/constructor#description)

Reply:
I recommend keeping it just for consistency between all bypass files.